connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -53,11 +53,8 @@ message SqlCommand {
   // (Required) SQL Query.
   string sql = 1;
 
-  // (Optional) A map of parameter names to string values that are parsed as
-  // SQL literal expressions. For example, map keys: "rank", "name", "birthdate";
-  // map values: "1", "'Steven'", "DATE'2023-03-21'". The fragments of string values
-  // belonged to SQL comments are skipped while parsing.
-  map<string, string> args = 2;
+  // (Optional) A map of parameter names to literal expressions.
+  map<string, Expression.Literal> args = 2;
```

Comment:
cc @grundprinzip are these protocol changes ok?

Reply:
No, this is an incomatible change.

Reply:
would be valid:

```suggestion
  map<string, string> args = 2;
  map<string, Expression.Literal> expr_args = 3;
```

so might be

```suggestion
  oneof args {
    map<string, string> args = 2;
    map<string, Expression.Literal> expr_args = 3;
  }
```

Please run 

```
buf breaking --against "https://github.com/apache/spark/archive/master.zip#strip_components=1,subdir=connector/connect/common/src/main"
```